File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.cpp
#include <string>
#include <vector>
#include <set>

using namespace std;

vector<int> solution(vector<int> numbers) {
    set<int> result;
    for (int i = 0; i < numbers.size()-1; i++) {
        for (int j = i + 1; j < numbers.size(); j++) {
            int sum = numbers[i] + numbers[j];
            result.insert(sum);
        }
    }
    vector<int> answer(result.begin(), result.end());
    
    return answer;
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    a = set()
    numbers.sort()
    for i in range (0, len(numbers)-1):
        for j in range (i+1, len(numbers)):
            a.add(numbers[i] + numbers[j])
    answer = list(a)
    answer.sort()
    return answer